modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/configuration/FFManagementToolbarConfigurationUtil.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.frontend.taglib.clay.internal.configuration;
+package com.liferay.frontend.taglib.clay.servlet.taglib.configuration;
```

Comment:
Just to confirm, are you really (as a team) ok with exposing a feature flag as API? Although cumbersome, in cases like this we create multiple feature flags (one per module) and we make sure we enable/disable them in sync (it's kind of a pain, but otherwise you'll be increasing the version numbers of your modules unnecessarily).

Reply:
Thanks for the review @adolfopa! I don't think we've had to face this case yet, but it makes sense to separate this into two different flags. Working on it :)